models/exam.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose');
+
+const Question = require('./question');
+
+const ExamSchema = new mongoose.Schema({
+    _examinerId : {
```

Comment:
Let's leave this as normal _id. You don't have to define it, it will be added automatically.

Reply:
Thats the id of the teacher or examiner setting the exam

Reply:
Oh, I see. Let's also store the id of the course.